plugins/embed-optimizer/hooks.php

```diff
@@ -153,3 +153,19 @@ function embed_optimizer_trigger_error( string $function_name, string $message,
 	}
 	wp_trigger_error( $function_name, $message, $error_level );
 }
+
+/**
+ * Displays the HTML generator tag for the Embed Optimizer plugin.
+ *
+ * See {@see 'wp_head'}.
+ *
+ * @since 0.1.0
+ */
+function embed_optimizer_render_generator() {
+	if (
+		defined( 'EMBED_OPTIMIZER_VERSION' )
+	) {
```

Comment:
This condition seems pointless if we always define the constant. But it is present in the other plugins.

Reply:
🤷🏼

Reply:
Maybe we should also check if `esc_attr()` exists and wrap it in a `try`/`catch` block then 😄